apple-silicon-support/packages/linux-asahi/default.nix

```diff
@@ -7,7 +7,7 @@
 , linuxPackagesFor
 , _4KBuild ? false
 , withRust ? false
-, kernelPatches ? [ ]
+, _kernelPatches ? [ ]
```

Comment:
Seemed like `callPackage` will fill this `kernelPatches` with... something. I get an `expected list got set` error before the change.